scripts/devops_tasks/common_tasks.py

```diff
@@ -21,32 +21,55 @@
 import re
 import pdb
 
-# ssumes the presence of setuptools
-from pkg_resources import parse_version, parse_requirements
+# Assumes the presence of setuptools
+from pkg_resources import parse_version, parse_requirements, Requirement
 
 # this assumes the presence of "packaging"
 from packaging.specifiers import SpecifierSet
 from packaging.version import Version
 
+from pip._internal.operations import freeze
+
+DEV_REQ_FILE = "dev_requirements.txt"
+NEW_DEV_REQ_FILE = "new_dev_requirements.txt"
 
 logging.getLogger().setLevel(logging.INFO)
 
 OMITTED_CI_PACKAGES = [
     "azure-mgmt-documentdb",
     "azure-servicemanagement-legacy",
-    "azure-mgmt-scheduler"
+    "azure-mgmt-scheduler",
 ]
 MANAGEMENT_PACKAGE_IDENTIFIERS = [
     "mgmt",
     "azure-cognitiveservices",
     "azure-servicefabric",
     "nspkg",
-    "azure-keyvault"
+    "azure-keyvault",
```

Comment:
Nit: `azure-keyvault` is placed in both `MANAGEMENT_PACKAGES` and `NON_MANAGEMENT`